### PR TITLE
Add Azure Maps Tiles provider

### PR DESCRIPTION
**Pull request by kopytjuk**
See https://docs.microsoft.com/en-us/rest/api/maps/renderv2/getmaptilepreview for details.

image

### CODE REVIEW

**Comment by brunob** [comment]
Seems good to me, need to be merged with squash, any thoughts @jieter ?

And thx @kopytjuk :)

**Comment by kopytjuk** [comment]
@brunob @jieter is there something I missed so far?

**Comment by brunob** [comment]
@kopytjuk nope, just waiting @jieter thoughts on this, no hurry...

**Comment by jieter** [comment]
Github CI is not run automatically for new contributors ([info](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks), I just approved the CI run. The lint check returns some errors, can you have a look at them @kopytjuk?

File: leaflet-providers.js [hunk 1]
```diff
@@ -1045,6 +1045,50 @@
...
 				pistes: 'pistes',
 			}
+		},
+		AzureMaps: {
+			url: 
+				'https://atlas.microsoft.com/map/tile?api-version={apiVersion}'+
+				'&tilesetId={variant}&x={x}&y={y}&zoom={z}&language={language}'+
+				'&subscription-key={subscriptionKey}',
+			options: {
+				attribution: 'See https://docs.microsoft.com/de-de/rest/api/maps/renderv2/getmaptilepreview for details.',
```
Comment (jieter) [inline leaflet-providers.js:1055]: Probably better to use an English locale in this url (en-en)?

**Review by jieter — APPROVED** [review]

**Comment by jieter** [comment]
@kopytjuk thanks, merged

**Pull request merged**